### PR TITLE
MCAT-23

### PR DESCRIPTION
**Pull request by VasiliyKu**
Added onboarding tests

### CODE REVIEW

**Comment by egorvas** [comment]
@VasiliyKu could you please add all targer folder to gitignore file

File: src/test/java/Temp.java [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+import io.appium.java_client.MobileDriver;
+import io.appium.java_client.pagefactory.iOSFindBy;
 import org.junit.*;
 
 /**
  * Created by VasilyKuznetsov on 29/06/16.
  */
 public class Temp extends Rules {
 
+    @iOSFindBy(name = "Test")
```
Comment (egorvas) [inline src/test/java/Temp.java]: Sounds like a bad idea to use FindBy annotation not in Page factory object? Why do you use it?

File: src/test/java/Temp.java [hunk 2]
```diff
@@ -23,7 +29,7 @@ public static void setup() throws Exception {
     @Before
```
Comment (egorvas) [inline src/test/java/Temp.java]: Why do you need this code?

File: src/test/java/ProfileEditSanityTests.java [hunk 3]
```diff
@@ -14,6 +14,8 @@
  */
 public class ProfileEditSanityTests extends Rules {
 
+
+
     @BeforeClass
```
Comment (egorvas) [inline src/test/java/ProfileEditSanityTests.java]: Why you wouldn't like to use it in the parent class?

File: src/test/java/Temp.java (hunk 1 above)
Comment (VasiliyKu) [inline src/test/java/Temp.java]: Thats my temp class for experiments, so disregard it please

File: src/test/java/Temp.java (hunk 2 above)
Comment (VasiliyKu) [inline src/test/java/Temp.java]: To use something before each test

File: src/test/java/ProfileEditSanityTests.java (hunk 3 above)
Comment (VasiliyKu) [inline src/test/java/ProfileEditSanityTests.java]: You mean to put it into Rules class?

**Pull request merged**